crypto/bio/bss_mem.c

```diff
@@ -299,7 +299,7 @@ static long mem_ctrl(BIO *b, int cmd, long num, void *ptr)
         ret = (long)bm->length;
         if (ptr != NULL) {
             pptr = (char **)ptr;
-            *pptr = (char *)&(bm->data[0]);
+            *pptr = (char *)(bm->data);
```

Comment:
Probably should drop the () around the pointer.

Reply:
Hmm..I think it is clearer with?